atlas_admin/templates/instances/sitestats.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block title %}{%- if themeName -%}{{ themeName }}{%- elif nodeCount -%}{{ nodeCount }}{%-  else -%}All{% endif %}
```

Comment:
Is `nodeCount` passed in to this template?

Reply:
not as yet; this is wip

Reply:
Let's remove unused code in that case

Reply:
done